docs/README.md

```diff
@@ -28,26 +28,9 @@ This guide uses some terminology that you have to know. Don't worry, it's easy:
   Brazilian Portuguese differs from the rest and it's called *pt_br*
   for the Git repo and *pt-br* for the mailing list suffix.
 
-## Table of Contents
-- [Overview of the build system](overview.md)
-- [Setting up a local build environment](local-setup.md)
-- [Editing the PHP Manual](editing.md)
-- [Documentation structure](structure.md)
-- [Style guidelines](style.md)
-- [Coding standard for examples](cs-for-examples.md)
-- [Translating documentation](translating.md)
-- [Joining the team](joining.md)
-
-## Appendices
-- [FAQ](faq.md)
-- [Tools](tools.md)
-- [Frequently used tags and entities](tags-and-entities.md)
-- [User Note Editing Guidelines](user-notes.md)
-- [How the released versions are built](public-builds.md)
-
 ## Feedback
 
 You can report problems or make contributions to this guide by using the
-"Edit this page" or "Report a problem" links in the header of each page
-at [the online version of this documentation](https://doc.php.net/tutorial/),
+"Edit this page" or "Report a problem" links in the sidebar of each page
+at [the online version of this documentation](https://doc.php.net/guide/),
```

Comment:
Isn't the current wording a tad odd?
```suggestion
"Edit this page" or "Report a problem" links in the sidebar of each page
of [online version of this documentation](https://doc.php.net/guide/),
```

Reply:
'of' is probably better than 'at', but we still need the 'the' in 'the online version'.